invenio_rdm_records/config.py

```diff
@@ -120,6 +120,87 @@ def _(x):
 FILES_REST_PERMISSION_FACTORY = record_files_permission_factory
 """Set default files permission factory."""
 
+# Invenio-IIIF
+# =================
+# See https://invenio-iiif.readthedocs.io/en/latest/configuration.html
+
+IIIF_PREVIEW_TEMPLATE = "invenio_app_rdm/iiif_preview.html"
+"""Template for IIIF image preview."""
+
+# Invenio-Previewer
+# =================
+# See https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/config.py  # noqa
+
+PREVIEWER_PREFERENCE = [
+    'csv_dthreejs',
+    'iiif_image',
+    'simple_image',
+    'json_prismjs',
+    'xml_prismjs',
+    'mistune',
+    'pdfjs',
+    'ipynb',
+    'zip',
+]
+"""Preferred previewers."""
+
+# Invenio-Records-UI
+# ==================
+# See https://invenio-records-ui.readthedocs.io/en/latest/configuration.html
+
+RECORDS_UI_ENDPOINTS = {
+    'recid': {
+        'pid_type': 'recid',
+        'record_class': 'invenio_records_files.api:Record',
+        'route': '/records/<pid_value>',
+        'template': 'invenio_rdm_records/record_landing_page.html'
+    },
+    'recid_files': {
+        'pid_type': 'recid',
+        'record_class': 'invenio_records_files.api:Record',
+        'route': '/records/<pid_value>/files/<path:filename>',
+        'view_imp': 'invenio_records_files.utils.file_download_ui',
+    },
+    'recid_previewer': {
+        'pid_type': 'recid',
+        'record_class': 'invenio_records_files.api:Record',
+        'route': '/records/<pid_value>/preview/<path:filename>',
+        'view_imp': 'invenio_previewer.views.preview',
+    },
+}
+
+#: Allow list of contributor types.
+RECORD_CONTRIBUTOR_TYPES = [
```

Comment:
Are these vocabularies somehow?

Reply:
As talked IRL. Yes, but to be decided how we handle them in the future :)